package.json

```diff
@@ -52,9 +58,8 @@
     "react-markdown": "^8.0.4",
     "react-player": "^2.11.0",
     "react-syntax-highlighter": "^15.6.1",
-    "remark-gfm": "^3.0.1",
+    "remark-gfm": "^4.0.1",
     "sass": "^1.81.0",
-    "tailwind": "^4.0.0",
```

Comment:
this is NOT [tailwindcss](https://www.npmjs.com/package/tailwindcss). apparently is a package related to event sourcing our something?

Anyways, it's unmaintained, and also it was blocking upgrade to node 22, because one of its dependencies (amqplib) needed npm lower than 9, which does not exist on node 22.